spec/package_spec.rb

```diff
@@ -293,6 +287,11 @@ def build_app_source_with_files(files)
     it 'ignores "description"' do
       expect(subject['description']).to be nil
     end
+
+    it 'preserves the rest' do
+      puts subject
+      expect(subject['everything_else']).to eq 'preserved'
```

Comment:
Fetch gives us more info and thus is favorable in debugging. 

For example, if assertion fails and tells us it expects hash[key] to be something and gets nil, we don't know if key is not in hash or the value of key in hash is nil. 

Like Tim mentioned above, fetch is always preferred unless a missing key is expected.